gen/generator.go

```diff
@@ -318,6 +319,54 @@ func (g *generator) recordGenDeclNames(d *ast.GenDecl) (conflict bool, err error
 	return false, nil
 }
 
+func (g *generator) declare(ignoreConflicts bool, s string, data interface{}, opts ...TemplateOption) error {
```

Comment:
Did this need to be moved?

Reply:
The comment was supposed to be for `DeclareFromTemplate`, but it was on `declare` - I could move it back if needed.

Reply:
Ah, I wonder how that happened. No, this is fine.